flask_iiif/api.py

```diff
@@ -134,6 +136,8 @@ def resize(self, dimensions, resample=Image.NEAREST):
 
         """
         real_width, real_height = self.image.size
+        if resample is None:
+            resample = import_string(current_app.config['IIIF_RESAMPLING'])
```

Comment:
Can you check if it is string type before importing it? Then you can define the value directly in `config.py` as `Image.NEAREST` (`from PIL import Image`).